index.html

```diff
@@ -222,6 +243,14 @@
       </dl>
 
       </section>
+
+      <section>
+        <h3>Providing outgoing payment responses</h3>
+
+        <p>When the <a>user accepts the payment request algorithm</a> runs in the context of a
+        payment app supporting basic-card, the payment app must provide back the relevant data in a
+        <code>BasicCardResponse</code> dictionary.</p>
```

Comment:
Is there a chance that converting from a random app to a BasicCardResponse would fail? If so, should we reject?